app/templates/base.html

```diff
@@ -1,68 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     {% block head %}
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
         integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- Tailwind CSS -->
+    <link href="https://unpkg.com/tailwindcss@1.9.5/dist/tailwind.min.css" rel="stylesheet">
```

Comment:
We don't use Tailwind in this project. 
We won't add it 'cause it will make a mess of a code (Bootstrap + Tailwind mishmash) and will make the page much heavier than it should be. :(

Reply:
No issue, I'll remove it

Reply:
By the way, I will remove Bootstrap before removing Tailwind :)
![image](https://user-images.githubusercontent.com/32178925/107156254-f3fdde80-6985-11eb-8902-60127cf652c5.png)

Reply:
That's almost a valid point, but `tailwind` is harder to work with if you don't use React/Vue/Angular, especially for new users, and will create a bloated code which will load slower. ^^"

If I have to choose only one of them (and that's the case) I would take Bootstrap

Reply:
@yammesicka - done - removed tailwind :)